vm/src/frame.rs

```diff
@@ -39,7 +38,6 @@ enum Block {
     },
 }
 
-#[derive(Clone)]
```

Comment:
Ow yeah!

Reply:
It doesn't work though.

I'll guess I'll have to split the frame up someone so we don't have to keep mutable references to it around.

Reply:
Yes, I believe so too, and we should probably use more of the PyObject and PyFrame types, which are references (`Rc<RefCell<>>`), and we can borrow them mutably or immutably.